docs/developer/workflows.md

```diff
@@ -9,3 +9,12 @@ To run the analysis.
 ```
 sh run.sh make excluded-xrefs-in-mondo
 ```
+
+## Mapping progress
```

Comment:
@matentzn Is this documentation sufficient for this feature?

Reply:
This is good but remember to document in the `help` goal in the mondo-ingest.Makefile.

Reply:
Oh right! Thanks for the reminder. I'll do that. Should this only be done for `.PHONY` goals or other goals?